packages/components/psammead-podcast-promo/src/components/card-link.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+import { C_PEBBLE } from '@bbc/psammead-styles/colours';
+
+const CardLink = styled.a`
+  color: unset;
+  :before {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    content: '';
+    overflow: hidden;
+    z-index: 1;
+  }
+  &:visited {
+    .podcast-promo__visited {
```

Comment:
should it be `.podcast-promo--visited` since it's a modifier class?

Reply:
👍 Updated